engine/src/cmd/collide2/OPC_HybridModel.cpp

```diff
@@ -301,11 +301,11 @@ bool HybridModel::Build(const OPCODECREATE& create)
  *	\return		amount of bytes used
  */
 ///////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
-udword HybridModel::GetUsedBytes() const
+ice_udword HybridModel::GetUsedBytes() const
```

Comment:
Hmm. This method isn't correct. I'll have to revisit it.

Reply:
If you're not going to do so before this gets merged, then please file an issue so it doesn't get forgotten

Reply:
Filed as #488